assets/animations.js

```diff
@@ -1,7 +1,9 @@
 const SCROLL_ANIMATION_TRIGGER_CLASSNAME = 'scroll-trigger';
 const SCROLL_ANIMATION_OFFSCREEN_CLASSNAME = 'scroll-trigger--offscreen';
+const SCROLL_ZOOM_IN_TRIGGER_CLASSNAME = 'animate--zoom-in';
```

Comment:
> Also, the effect doesn't seem to kick in until I've hit the save button.

So this is something that we should think about in general. This is the case for a few of our JS dependant features.
Tweaking the settings of a section only reloads that section. So any "global" JS that is loaded with the whole theme isn't re run and therefore not picking up the new elements to animate until the whole theme is reloaded (save or navigate to a different page). 

If we had `animations.js` loaded per section then it would re run but it would create a lot of calls for the same file over and over (say if you had 4 image banner sections for example).

Reply:
I agree, we should prioritize a refactor for that in the near future if at all possible.